packages/nano-router-plugin/src/index.js

```diff
@@ -6,13 +6,12 @@ export const nanoRouterPlugin = ({ routes, history }) => (store) => {
   const updateRouting = () => {
     store.dispatch({
       payload: {
-        route: router.route,
-        location: router.location,
-        params: router.params,
-        queryParams: router.queryParams,
-      },
-      apply: (cache, { payload }) => {
```

Comment:
Ah, so apply is optional to care for situation where it's a static payload?

Reply:
No apply is completely gone :-)